### ISSUE TITLE
Offer a command to open a tree/list's context menu

### ISSUE THREAD

**Issue by dakaraphi** [microsoft/vscode#20814]
The context menu when invoked using a key binding only works in an editor.  If something else has focus like the file explorer and you invoke the context menu, you will get the context menu of the editor and not the item in focus for the file explorer.

vscode: 1.9.1
OSX 10.11.6

**Comment by bpasero** [comment]
@dakaraphi what keybinding are you using?

**Comment by dakaraphi** [comment]
@bpasero ctrl+`

**Comment by bpasero** [comment]
@joaomoreno can you reproduce?

**Comment by joaomoreno** [comment]
@dakaraphi Did you configure <kbd>Ctrl `</kbd> to execute the command `editor.action.showContextMenu`?

**Comment by dakaraphi** [comment]
@joaomoreno yes.

**Comment by joaomoreno** [comment]
The command is meant for showing the editor's context menu, nothing more. We don't have commands to show the context menu of everything else.

**Comment by dakaraphi** [comment]
@joaomoreno ok, then can this be a feature request.  Other editors offer key bindings for the context menu for the item in focus.  The more we can use the keyboard and not touch the mouse, the faster the workflow can be.

**Comment by dakaraphi** [comment]
Note, for clarity, ideally there should be only 1 command that operates on context for anywhere.  This can be a new command, so existing behavior is preserved; however, it would be cumbersome to have to manage different commands based on the type of panel that is active.

**Comment by bpasero** [comment]
Yeah, this goes in line with our newly introduced list commands: https://github.com/Microsoft/vscode-docs/blob/vnext/release-notes/v1_10.md#configurable-treelist-keybindings

**Comment by isidorn** [comment]
I think adding a command that shows a tree context menu makes perfect sense from the accessibility point of view.
Asssigning myself and June.
I plan to put it in the `listCommands.ts`

**Comment by isidorn** [comment]
After investigating this actually works already. It is just not changable
`Shift + F10`
This is our standard keybinding, also in the editor.

Thus closing this as it works.
If you still want this configurable let me know and I can reopen, though it is not an accessibility issue then.
